### DISCUSSION TITLE
Submitting multiple forms at once

### DISCUSSION THREAD

**Discussion by umutyorulmaz** [lowcoder-org/lowcoder#1398] [General]
I have a tab container with multiple tabs, each tab has a form in it. I want to submit all of the forms filled from the last tab using its submit button, how can I do that? Thank you.

**Comment by placidic** [comment]
There's a few ways, but on the last tab you could set the action of the submit button, to trigger a JavaScript query object containing code to pull the values from each of the form controls and then run the appropriate data query you want to run with the results. In your data query you would set a placeholder object to use as a parameter as described here: 

https://docs.lowcoder.cloud/lowcoder-documentation/business-logic-in-apps/write-javascript/javascript-query

Your query could look something along the lines of:

```
let payload = {
name: form1.value.name,
email: form2.value.email
}

targetQuery.run({payload: payload});

```

**Comment by FalkWolsky** [comment]
Yes, that it pretty much it. You would need to collect all the form data, which is accessible via JavaScript. Then you can merge it into the format your backend need - and then send it.
